feature/amazon/src/main/java/com/revenuecat/purchases/amazon/ISO3166Alpha2ToISO42170Converter.kt

```diff
@@ -0,0 +1,254 @@
+        "LS" to "ZAR",
+        "LR" to "LRD",
+        "LY" to "LYD",
+        "LI" to "CHF",
+        "LT" to "EUR",
+        "LU" to "EUR",
+        "MO" to "MOP",
+        "MK" to "MKD",
+        "MG" to "MGA",
+        "MW" to "MWK",
+        "MY" to "MYR",
+        "MV" to "MVR",
+        "ML" to "XOF",
+        "MT" to "EUR",
+        "MH" to "USD",
+        "MQ" to "EUR",
+        "MR" to "MRO",
+        "MU" to "MUR",
+        "YT" to "EUR",
+        "MX" to "MXN",
+        "FM" to "USD",
+        "MD" to "MDL",
+        "MC" to "EUR",
+        "MN" to "MNT",
+        "ME" to "EUR",
+        "MS" to "XCD",
+        "MA" to "MAD",
+        "MZ" to "MZN",
+        "MM" to "MMK",
+        "NA" to "ZAR",
+        "NR" to "AUD",
+        "NP" to "NPR",
+        "NL" to "EUR",
+        "NC" to "XPF",
+        "NZ" to "NZD",
+        "NI" to "NIO",
+        "NE" to "XOF",
+        "NG" to "NGN",
+        "NU" to "NZD",
+        "NF" to "AUD",
+        "MP" to "USD",
+        "NO" to "NOK",
+        "OM" to "OMR",
+        "PK" to "PKR",
+        "PW" to "USD",
+        "PA" to "USD",
+        "PG" to "PGK",
+        "PY" to "PYG",
+        "PE" to "PEN",
+        "PH" to "PHP",
+        "PN" to "NZD",
+        "PL" to "PLN",
+        "PT" to "EUR",
+        "PR" to "USD",
+        "QA" to "QAR",
+        "RO" to "RON",
+        "RU" to "RUB",
+        "RW" to "RWF",
+        "RE" to "EUR",
+        "BL" to "EUR",
+        "SH" to "SHP",
+        "KN" to "XCD",
+        "LC" to "XCD",
+        "MF" to "EUR",
+        "PM" to "EUR",
+        "VC" to "XCD",
+        "WS" to "WST",
+        "SM" to "EUR",
+        "ST" to "STD",
+        "SA" to "SAR",
+        "SN" to "XOF",
+        "RS" to "RSD",
+        "SC" to "SCR",
+        "SL" to "SLL",
+        "SG" to "SGD",
+        "SX" to "ANG",
+        "SK" to "EUR",
+        "SI" to "EUR",
+        "SB" to "SBD",
+        "SO" to "SOS",
+        "ZA" to "ZAR",
+        "SS" to "SSP",
+        "ES" to "EUR",
+        "LK" to "LKR",
+        "SD" to "SDG",
+        "SR" to "SRD",
+        "SJ" to "NOK",
+        "SZ" to "SZL",
+        "SE" to "SEK",
+        "CH" to "CHF",
+        "SY" to "SYP",
+        "TW" to "TWD",
+        "TJ" to "TJS",
+        "TZ" to "TZS",
+        "TH" to "THB",
+        "TL" to "USD",
+        "TG" to "XOF",
+        "TK" to "NZD",
+        "TO" to "TOP",
+        "TT" to "TTD",
+        "TN" to "TND",
+        "TR" to "TRY",
+        "TM" to "TMT",
+        "TC" to "USD",
+        "TV" to "AUD",
+        "UG" to "UGX",
+        "UA" to "UAH",
+        "AE" to "AED",
+        "GB" to "GBP",
+        "US" to "USD",
+        "UM" to "USD",
+        "UY" to "UYU",
+        "UZ" to "UZS",
+        "VU" to "VUV",
+        "VE" to "VEF",
+        "VN" to "VND",
+        "VG" to "USD",
+        "VI" to "USD",
+        "WF" to "XPF",
+        "EH" to "MAD",
+        "YE" to "YER",
+        "ZM" to "ZMW",
+        "ZW" to "ZWL",
+        "AX" to "EUR"
+    )
+}
```

Comment:
Unity IAP does it this way, I think this is the best without introducing external dependencies.

Reply:
Can we add something in docs here or put it into Notion about how got this mapping? (For if we need to update it in the future 😉 )

Reply:
Yes, I want to do a write up about this. I can add where we got it from there